packages/xo-web/src/common/xo/copy-vms-modal/index.js

```diff
@@ -124,8 +125,15 @@ class CopyVmsModalBody extends BaseComponent {
               <SelectSr
                 disabled={copyMode !== 'fullCopy'}
                 onChange={this.linkState('sr')}
+                predicate={sr =>
+                  process.env.XOA_PLAN > 2 ||
+                  every(resolvedVms, { $poolId: sr.$pool })
+                }
                 value={sr}
               />
+              {(getXoaPlan() === 'Free' || getXoaPlan() === 'Starter') && (
```

Comment:
You can create a const and mutualize it instead because the plan is static.

```js
const isPlanHigherThanStarter = getXoaPan().value > Starter.value
```